Case Study/vehicalDekho/helper/VehicleHelper.java

```diff
@@ -0,0 +1,54 @@
+package helper;
+
+import model.Bike;
+import model.Car;
+import model.Vehicle;
+
+public class VehicleHelper {
+
+	public static Vehicle createVehicle(String vehicleType, String model,
+			String make) {
+
+		if (vehicleType.equals("car")) {
+			return createCar(model, make);
+
+		} else {
+			return createBike(model, make);
+
+		}
+
+	}
+
+	private static Vehicle createCar(String model, String make) {
```

Comment:
As the name clearly shows that the method creates car only therefore it can safely return "Car" instead of "Vehicle"